lib/modules/backupAndRestore.js

```diff
@@ -103,6 +103,34 @@ module.options = {
 		title: 'backupAndRestoreGoogleAccountTitle',
 		advanced: true,
 	},
+	webDAV: {
+		type: 'table',
+		addRowText: 'backupAndRestoreAddWebDAVAccount',
+		fields: [{
+			key: 'username',
+			name: 'backupAndRestoreUsername',
+			type: 'text',
+		}, {
+			key: 'password',
+			name: 'backupAndRestorePassword',
+			type: 'password',
+		}, {
+			key: 'url',
+			name: 'backupAndRestoreUrl',
+			type: 'text',
+		}, {
+			key: 'digest',
+			name: 'backupAndRestoreDigest',
+			type: 'boolean',
+			value: false,
+		}],
+		value: ([]: Array<[string, string, string, boolean]>),
+		description: 'backupAndRestoreWebDAVDesc',
+		title: 'backupAndRestoreWebDAVTitle',
+		onChange() {
+			window.alert('RES stores WebDAV credentials in plaintext.');
```

Comment:
Using `window.alert` in embedded cross-origin frames (which is the case for RES' option page) is [slated to be blocked](https://chromestatus.com/feature/5148698084376576).  As an alternative, you could use the `showNotification` function.

Reply:
Was not aware that the options page was a cross-origin frame. I'll change this ASAP.